client/my-sites/marketing/tools/index.tsx

```diff
@@ -92,7 +92,7 @@ export const MarketingTools: FunctionComponent< Props > = ( {
 					imagePath="/calypso/images/marketing/social-media-logos.svg"
 				>
 					<Button compact onClick={ handleStartSharingClick }>
-						{ translate( 'Start Sharing' ) }
+						{ translate( 'Start sharing' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.